### DISCUSSION TITLE
Continuation of time series

### DISCUSSION THREAD

**Discussion by zoyafuso-NOAA** [afsc-gap-products/StationAllocationAIGOA#8] [General]
@Lewis-Barnett-NOAA @Ned-Laman-NOAA @MargaretSiple-NOAA 

**Do we need to recalculate the design based estimates (DBE) of abundance on years pre-2023 using the new 2023 STRS design?** 

- We do not need to recalculate the DBEs for years pre-2023 using the new STRS design. Those pre-2023 estimates are created with the design used during sampling and will not change. 
- From 2023 until the next design change, DBEs of abundance will be calculated using the new 2023 STRS design.
- The implication of this is that the survey data products we produce in 2023 will be in a different format to what has been done historically, potentially altering the data workflow that has been created to produce those estimates for reports. 

Just wanted to have this response in a centralized place.

**Comment by Ned-Laman-NOAA** [comment]
I agree with all of Zack's bullet points here, but want to shade in a related topic.

We will have to maintain the ability to recalculate old biomass estimates from the historic design using the historic code. E.g., if through usage errors are pointed out that need to be corrected in the 2021 GOA data, those changes might require that we rerun biomass in the 2021 flavor. In the future we envision, the tool to do this would be integrated into R and would replicate the old school estimates if for no other reason than to move maintenance and tools all into one location and possibly one process...thoughts?

**Comment by MargaretSiple-NOAA** [comment]
Yes; the total biomasses will be continuous across the whole survey time series (a strength of moving from one STRS design to another), but the biomass by stratum will not be continuous. I think this just means no one will be able to make, say, a time series of biomass in stratum X over time without post-stratifying the old survey data. 

Re: Ned's comment, I think the calculations will be the same, it's just that the stratum areas will change. So the script would only need to use a different version of the `GOA_STRATA` table. I think.

**Comment by zoyafuso-NOAA** [comment]
Yes to all of Megsie's point; gulf-wide biomass times series will be continuous but a biomass time series for a particular stratum will not be.  The calculations will/should be the same. 

I am very much for having one general process that calculates the DBEs of any STRS design (AI, historical GOA design, new GOA design). It's also the one of the topics that we discussed in our openscapes team (Zack, Sarah, Lukas, and Duane) meeting. So going off yesterday's GOA team meeting, that's also me volunteering for that task group.

**Reply by zoyafuso-NOAA** [reply]
Bering Sea folks also provide estimates by stratum so that one process can also apply there as well.
